README.md

```diff
@@ -80,6 +80,10 @@ Basically, if your university calls it AI, it lives here.
 * [KANN][327] - Two-file ANN library. [``MIT``][MIT]
 * [LibDEEP][477] - Deep learning library. [``BSD-3-Clause``][BSD-3-Clause]
 
+## Benchmarking ##
```

Comment:
There should also be a link from the contents section to this one. It might be worth using the start of this section to explain the difference between profiling and benchmarking (thus justifying the two different sections) as well.